src/components/MyWallets/MyWallets.tsx

```diff
@@ -10,31 +12,37 @@ import {
   WalletBalances
 } from './styles';
 
-const MyWallets = () => (
-  <MyWalletsContainer>
-    <Sidebar>
-      <WalletList />
-      <TotalBalance />
-      <ManageAccountLink
-        href={process.env.REACT_APP_WEBWALLET_URL}
-        target="_blank"
-      >
-        Manage Wallets
-      </ManageAccountLink>
-    </Sidebar>
-    <WalletBalances>
-      <WalletBalanceListHeader>
-        <thead>
-          <tr>
-            <HeaderCell w="20%">Asset</HeaderCell>
-            <HeaderCell w="40%">Base currency</HeaderCell>
-            <HeaderCell w="40%">Balance</HeaderCell>
-          </tr>
-        </thead>
-      </WalletBalanceListHeader>
-      <WalletBalanceList />
-    </WalletBalances>
-  </MyWalletsContainer>
-);
+interface MyWalletsProps {
+  addModal: any;
+}
+
+const MyWallets: React.SFC<MyWalletsProps> = ({addModal}) => {
+  const handleManageWallets = () =>
+    addModal(ModalMessages.manageWallets, null, null, Types.ManageFunds);
```

Comment:
you may and should inject it directly to the _Manage funds_ button

Reply:
and leave this component as it was before, cause it's just combines connected components